Branch-SDK/Branch-SDK/Requests/BranchOpenRequest.m

```diff
@@ -82,6 +82,7 @@ - (void)makeRequest:(BNCServerInterface *)serverInterface key:(NSString *)key ca
 
 - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
     if (error) {
+        [BranchOpenRequest releaseOpenResponseLock];    
```

Comment:
Does this cover the retry case? Where a request fails & returns a non-nil error, then triggers a retry call? Will the queue be re-incremented?